### PR TITLE
Add CobaltCapabilities to CheckFileInfo

### PR DESCRIPTION
**Pull request by xiaohya-msft**
An array of strings named CobaltCapabilities can be returned in the response of CheckFileInfo. Add this to the document

### CODE REVIEW

File: docs/rest/src/files/CheckFileInfo.rst [hunk 1]
```diff
@@ -643,6 +643,15 @@ because they are pending deprecation or they are designated for future features
         ..  note::
             This does not mean that the user doesn't have rights to edit the file. Hosts should use the
             :term:`UserCanWrite` property for that purpose.
+    
+    CobaltCapabilities
+        A **array** of strings that contains the Cobalt capabilities supported by the host.
```
Comment (tylerbutler) [inline docs/rest/src/files/CheckFileInfo.rst]: nit: ``**array of strings**``

File: docs/rest/src/files/CheckFileInfo.rst [hunk 2]
```diff
@@ -643,6 +643,15 @@ because they are pending deprecation or they are designated for future features
...
+
```
Comment (tylerbutler) [inline docs/rest/src/files/CheckFileInfo.rst]: I recommend adding something like,
```
If :term:`SupportsCobalt` is set to `false`, this property must be ignored by the WOPI client.
Any value other than those listed below must be ignored by the WOPI client.
```

File: docs/rest/src/files/CheckFileInfo.rst [hunk 3]
```diff
@@ -643,6 +643,15 @@ because they are pending deprecation or they are designated for future features
...
+        Possible Values:
+
+        DownloadStreaming
+            This type of CobaltCapabilities indicates the host supports cobalt streaming for download
```
Comment (tylerbutler) [inline docs/rest/src/files/CheckFileInfo.rst]: nit: caitalize Cobalt.

File: docs/rest/src/files/CheckFileInfo.rst (hunk 2 above)
Comment (xiaohya-msft) [reply docs/rest/src/files/CheckFileInfo.rst]: Thanks! Added

**Pull request merged**